tiledb/ml/models/pytorch.py

```diff
@@ -173,16 +172,18 @@ def preview(self) -> str:
         """
         return str(self.artifact) if self.artifact else ""
 
-    def _create_array_internal(self, **kwargs: Any) -> None:
+    def __create_array(self, **kwargs: Any) -> None:
```

Comment:
Since this is private it doesn't have to have the same signature across all artifact classes, so in this case the signature can be simplified to `def __create_array(self, serialized_model_info)`. Likewise for other artifacts.